Dockerfile.gpu

```diff
@@ -1,4 +1,24 @@
-FROM nvidia/cuda:12.4.1-devel-ubuntu20.04
+FROM nvidia/cuda:12.4.1-devel-ubuntu22.04 as builder
+
+# FIXME: should be able to find this dynamically
+ARG GPU_ARCH=89
+
+# GPU_ARCH and USEGPU environment variable used by build_proc.sh
+ENV FFTW_LIB=/usr/lib/x86_64-linux-gnu/libfftw3f.a
+ENV GPU_ARCH=${GPU_ARCH}
+ENV USEGPU=true
+ENV DEBIAN_FRONTEND=noninteractive
+
+# FIXME: can remove git after switch back to released version of back-projection
+RUN apt-get update && apt-get install -y --no-install-recommends unzip vim curl git build-essential gfortran libfftw3-dev && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
Why `rm -rf /var/lib/apt/lists/*`?

Reply:
This removes unneeded `apt` metadata that serves no purpose other than bloating the container.